src/sentry/static/sentry/app/views/performance/transactionSummary/durationChart.tsx

```diff
@@ -48,17 +51,45 @@ const YAXIS_VALUES = ['p50()', 'p75()', 'p95()', 'p99()', 'p100()'];
  * percentiles over the past 7 days
  */
 class DurationChart extends React.Component<Props> {
+  handleLegendSelectChanged = (legendChange, _event) => {
```

Comment:
```suggestion
  handleLegendSelectChanged = (legendChange) => {
```

No need to bind a terminal parameter if you aren't using it.